notifier/notifier.go

```diff
@@ -623,10 +685,14 @@ func (n *Manager) sendOne(ctx context.Context, c *http.Client, url string, b []b
 	return nil
 }
 
-// Stop shuts down the notification handler.
+// Stop signals the notification manager to shut down and immediately returns.
+//
+// Run will return once the notification manager has successfully shut down.
+//
+// The manager will optionally drain any queued notifications before shutting down.
 func (n *Manager) Stop() {
```

Comment:
There's a behaviour change that's worth to be documented. Previously, if you called `Stop()` twice it was fine. Now it will panic.

Reply:
Good catch. Rather than documenting the behaviour, I've made it safe to call `Stop` multiple times in 6433d28.